monogen/testapp/docs/static/openapi.yml

```
@@ -0,0 +1 @@
+{"id":"github.com/testapp/testapp","consumes":["application/json"],"produces":["application/json"],"swagger":"2.0","info":{"description":"Chain github.com/testapp/testapp REST API","title":"HTTP API Console","contact":{"name":"github.com/testapp/testapp"},"version":"version not set"},"paths":{},"definitions":{"google.protobuf.Any":{"type":"object","properties":{"@type":{"type":"string"}},"additionalProperties":{}},"google.rpc.Status":{"type":"object","properties":{"code":{"type":"integer","format":"int32"},"details":{"type":"array","items":{"type":"object","$ref":"#/definitions/google.protobuf.Any"}},"message":{"type":"string"}}}}}
```

yamllint

[warning] too few spaces after comma

(commas)